### PR TITLE
[MBL-15671][Student][Teacher] PSPDFKit update

### PR DESCRIPTION
**Pull request by tamaskozmer**
Test plan: Verify that every PSPDFKit functionality works in the app as intended:
- Annotating PDF submission types and student annotation submission types as a teacher.
- Opening PDF files as a teacher.
- Annotating PDF files for student annotation submission type and submitting the assignment as a student.
- Opening PDF files as a student (here you can add annotations, search in the file, change different settings about how the PDF is shown, these should be tested as well).
- Compare all functionality with the current app available in the Play Store.

There were also some proguard issues so the best way to test this build would be to build it locally while changing the `minifyEnabled` and the `shrinkResources` properties to `true` in the debug config of the `build.gradle` file of the student and teacher app.

refs: MBL-15671
affects: Teacher, Student
release note: none

### CODE REVIEW

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Review by kdeakinstructure — APPROVED** [review]
QA 👍

File: apps/student/src/main/res/menu/pspdf_activity_menu.xml [hunk 1]
```diff
@@ -23,6 +23,7 @@
         android:id="@+id/mainMenu"
         android:icon="@drawable/ic_overflow_white_18dp"
         android:title="@string/uploadToCanvas"
+        app:iconTint="#b3e5fc"
```
Comment (hermannakos) [inline apps/student/src/main/res/menu/pspdf_activity_menu.xml:26]: Should we use a color res for this?

**Review by hermannakos — APPROVED** [review]

File: apps/teacher/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -21,6 +21,7 @@
     <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
```
Comment (stephaniegilbert944) [inline apps/teacher/src/main/AndroidManifest.xml:21]: Hi

**Pull request merged**